lib/index.js

```diff
@@ -319,13 +379,21 @@ module.exports = biojsvispca = function(init_options){
     // setup the SVG. We do this inside the d3.tsv as we want to keep everything in the same place
     // and inside the d3.tsv we get the data ready to go (called options.data here)
     var svg = d3.select(options.target).append("svg")
-        .attr("width", full_width)
-        .attr("height",full_height)
+        .attr("width", scatterWidth)
+        .attr("height",scatterHeight)
+      .append("g")
+        // this is just to move the picture down to the right margin length
+        .attr("transform", "translate(" + left + "," + top + ")");
+
+    var svg2 = d3.select(options.target).append("svg")
```

Comment:
The zoom function works in the whole svg area.

If we put 2 plots in one svg element, we will find that the circles in the scatter plot can reach the bar chart area when zooming in.

So, I use one svg for scatter plot, one svg for bar chart to make sure that the zoom function only works in the scatter plot.